source/slang/slang-ast-type.h

```diff
@@ -97,6 +97,11 @@ class BasicExpressionType : public ArithmeticExpressionType
     }
 };
 
+class IDefaultInitializable : public DeclRefType
```

Comment:
`IDefaultInitializableType`

Reply:
It doesn't seem like this is actually used by any front end logic? Can it be removed?